### ISSUE TITLE
[Bug] Expander - content is moving during the expand/collapse animation

### ISSUE THREAD

**Issue by Reveon** [xamarin/Xamarin.Forms#10513]
### Description
Content is moving when the Expander is running collapse or expand animation. This has impact especially on images where different aspect settings cause different issues. It also seems to cut label in the bottom. See the gif animation bellow.

### Steps to Reproduce

1. Run the repro sample
2. Expand and collapse included expanders

### Expected Behavior
Content should always stay in the same location during the expand/collapse animation and its size should stay the same.

### Actual Behavior
Content is moving/jumping while the animation is running. Images are getting smaller/larger.

### Basic Information

- Version with issue: 4.6.0.726 
- Last known good version: -
- IDE: VS 2019 16.5.4
- Platform Target Frameworks: 
  - Android: Android 10 emulator

### Screenshots

![Expander](https://user-images.githubusercontent.com/29383312/80641824-bf0f4380-8a65-11ea-8995-ad0fb175e9e8.gif)

### Reproduction Link

[ExpanderContentMoving.zip](https://github.com/xamarin/Xamarin.Forms/files/4554319/ExpanderContentMoving.zip)

**Comment by rmarinho** [comment]
hey @AndreiMisiukevich i think this might a enhancement no? 
Would be possible to size the items, and then expand them? or maybe fade them in ?

**Comment by AndreiMisiukevich** [comment]
hey @rmarinho 

Yes, definitely it might be. 
Need to try some approaches.

Fading in would be the easiest way, I believe, but not sure that it will look good (If I understood your idea correctly) :)

**Comment by Reveon** [comment]
I think that the content should just stay in the same position and keep its size because current behavior can lead to some really weird effects. Especially if there are more images with different aspect settings. I think that on iOS this was mostly fine.

**Comment by rmarinho** [comment]
Yeah i think we need to think a little bit on this, and see other examples maybe.

But for sure we would need some kind of `ExpandMode`

**Comment by AndreiMisiukevich** [comment]
@rmarinho please add "a/expander" label
